controllers/discordactions.js

```diff
@@ -122,8 +122,28 @@ const addGroupRoleToMember = async (req, res) => {
   }
 };
 
+/**
+ * Gets all group-roles
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const updateDiscordImageForVerification = async (req, res) => {
+  try {
+    const { id: userDiscordId } = req.params;
+    const discordAvatarUrl = await discordRolesModel.updateDiscordImageForVerification(userDiscordId);
```

Comment:
Instead of using the try-catch block around the current lines, it can be applied specifically around this line.

Reply:
Thats just one line await thats why felt unnecessary.

Reply:
Certainly, I kindly suggest that you consider incorporating this in the future.🙂